manifests/init.pp

```diff
@@ -7,17 +7,40 @@
 # Document parameters here.
 #
 # [*secret_key*]
-#   Your Takipi secret key
+#   Required. Your Takipi secret key.
 #
 # [*source*]
-#   Foreign url to fetch Takipi from. This is optional
+#   Optional. Foreign url to fetch Takipi from.
+#
+# [*manage_java*]
+#   Optional. Java agent only. Installs Java package.
+#
+# [*collector_host*]
+#   Required. .Net agent only. IP of remote collector.
```

Comment:
if you're working on the Puppet script, it's OK to finally brush it and make it speak "remote collector" first as that's our standard for Java these days anyway.